webview/src/setup/components/App.test.tsx

```diff
@@ -18,7 +18,6 @@ import { TooltipIconType } from '../../shared/components/sectionContainer/InfoTo
 import { setupReducers } from '../store'
 
 jest.mock('../../shared/api')
-jest.mock('../../shared/components/codeSlider/CodeSlider')
```

Comment:
[Q] @sroy3 is there a reason that this is under shared components? Shouldn't it be under the setup webview?

Reply:
I can't remember, but I think it might be because there was no data at all that would make it purely a setup component, and it could be reused anywhere if necessary. Components were still pretty big back then, so I saw the `shared` components as being the dumb components would go. A little like a component library. But it's not really the direction we are moving in at the moment. It's fine to move it.